test/smoke_test/bcda_client.go

```diff
@@ -40,8 +40,9 @@ func init() {
 	flag.StringVar(&clientSecret, "clientSecret", "", "client secret for retrieving an access token")
 	flag.StringVar(&apiHost, "host", "localhost:3000", "host to send requests to")
 	flag.StringVar(&proto, "proto", "http", "protocol to use")
-	flag.StringVar(&endpoint, "endpoint", "", "endpoint to test")
+	flag.StringVar(&resourceType, "resourceType", "", "resourceType to test")
 	flag.IntVar(&timeout, "timeout", 300, "amount of time to wait for file to be ready and downloaded.")
+	flag.StringVar(&endpointBase, "endpointBase", "", "base type of request endpoint")
```

Comment:
Can we keep the name `endpoint` instead of `endpointBase`?

Reply:
done